docs/Components/BaseComponent.md

```diff
@@ -34,3 +34,29 @@ print(base_component.children)
 print(base_component.wing1)
 >>> BaseComponent(name='wing1', type='wing', mass=1.1, children={}, external_forces={}, external_moments={})
 ```
+## angle_of_attack method
+This method calculates the angle of attack of the plane
+```python
+attached_component = AttachedComponent(
+    name='attached_component',
+    type='generic_attached_component',
+    mass=1.4,
+    relative_position=Vector2(-0.4, 0.1),
+    relative_angle=math.radians(9)
+)
+freebody_component = FreeBody(
+    name='component',
+    type='generic_component',
+    mass=3.4,
+    position_cg=Vector2(-0.7, 0.2),
+    pitch_rot_inertia=30.0,
+    ambient=Ambient()
+)
+
+print(freebody_component.angle_of_attack)
+>>> -0.7853981633974483
```

Comment:
It would be better to print the angle also in degrees, like `print(math.degrees(angle))`.